### PR TITLE
Release notes for September 2021 release

### PR DESCRIPTION
**Pull request by beccamc**
@zewditu Can I get more details on the consume code updates?

@LittleLittleCloud What other features are making the September release?

### CODE REVIEW

File: docs/release-notes/9.2021.md [hunk 1]
```diff
@@ -0,0 +1,18 @@
+### Improve consumption code 
+_Need to get more details from Zewditu here._ [#1605](https://github.com/dotnet/machinelearning-modelbuilder/issues/1605) [#1567](https://github.com/dotnet/machinelearning-modelbuilder/issues/1567) [#1419](https://github.com/dotnet/machinelearning-modelbuilder/issues/1419)
+
+### Usability 
+- SQL Server Authentication fix [#1540](https://github.com/dotnet/machinelearning-modelbuilder/issues/1540)
+- New Azure Workspace create [#1104](https://github.com/dotnet/machinelearning-modelbuilder/issues/1104), [#1492](https://github.com/dotnet/machinelearning-modelbuilder/issues/1492), [#1654](https://github.com/dotnet/machinelearning-modelbuilder/issues/1654)
+- GPU fix: Multiple versions of CUDA can be installed. Only requirment is Cuda 10.1 is installed and Cuda_Path_v10_1 is set. [#1642](https://github.com/dotnet/machinelearning-modelbuilder/issues/1642)
```
Comment (LittleLittleCloud) [inline docs/release-notes/9.2021.md:7]: We might need to update document as well

File: docs/release-notes/9.2021.md [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+
+### UI fixes
+- Replace Image Classification folder picker with file picker filtered to folders. [#394](https://github.com/dotnet/machinelearning-modelbuilder/issues/394)
+- Improved usability of Advanced Data Options. Increase size of the name column and add tooltip for long names. [#1534](https://github.com/dotnet/machinelearning-modelbuilder/issues/1534)
+- Rename confusing "Text classification" to "Data classification" [#1710](https://github.com/dotnet/machinelearning-modelbuilder/issues/1710)
+- Some text gets cut off in dark mode [#1695](https://github.com/dotnet/machinelearning-modelbuilder/issues/1695)
```
Comment (LittleLittleCloud) [inline docs/release-notes/9.2021.md:13]: This issue still exists

**Comment by LittleLittleCloud** [comment]
Do we want to include all fixed issues as well?

**Comment by zewditu** [comment]
Not too much on consuming page except we API   it is based on  .Net 6 previews' [Miminimal web API ](URL)

File: docs/release-notes/9.2021.md (hunk 1 above)
Comment (LittleLittleCloud) [reply docs/release-notes/9.2021.md:7]: @luisquintanilla In [install-dependencies](https://docs.microsoft.com/en-us/dotnet/machine-learning/how-to-guides/install-gpu-model-builder#install-dependencies) section

> Install CUDA v10.1. Make sure you install CUDA v10.1, not any other newer version. You cannot have multiple versions of CUDA installed.

We can remove `You cannot have multiple versions of CUDA installed.`

**Comment by beccamc** [comment]
@zewditu So for consumption code, those three bugs were fixed (see the release notes for numbers) and we updated the Web API to be based on .NET 6 Minimal Web API?

**Comment by beccamc** [comment]
I added all of XiaoYun's comments. @briacht I think we should talk about the removal of the Preview tag in the blog instead of including it here (or do both).

File: docs/release-notes/9.2021.md (hunk 1 above)
Comment (luisquintanilla) [reply docs/release-notes/9.2021.md]: @LittleLittleCloud does this look good?

**Comment by beccamc** [comment]
@zewditu They are in this PR. Look at the "Files changed" tab

File: docs/release-notes/9.2021.md (hunk 1 above)
Comment (LittleLittleCloud) [reply docs/release-notes/9.2021.md]: @luisquintanilla LGTM

**Comment by zewditu** [comment]
-  Improve the performance of Predict method in Consumption. cs  in the code behind ( [1567](https://github.com/dotnet/machinelearning-modelbuilder/issues/1567) ,  [1419](https://github.com/dotnet/machinelearning-modelbuilder/issues/1419) )

**Pull request merged**